01week/rockPaperScissors.js

```diff
@@ -9,10 +9,34 @@ const rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
+  hand1 = hand1.toLowerCase().trim();
+  hand2 = hand2.toLowerCase().trim();
+
+  if(hand1 === hand2){
+    return "It's a tie!"
+  }else if
+    (hand1 === "rock" && hand2 === "scissors"){
+    return "Hand one wins!"
+  }else if 
+    (hand1 === "scissors" && hand2 === "paper"){
+      return "Hand one wins!"
+    }else if
+      (hand1 === "paper" && hand2 === "rock"){
+        return "Hand one wins!"
+      }else if
+        (hand2 === "rock" && hand1 === "scissors"){
+          return "Hand two wins!"
+        }else if
+          (hand2 === "scissors" && hand1 === "paper"){
+            return "Hand two wins!"
+          }else
+          (hand2 === "paper" && hand2 === "rock")
+            return "Hand two wins!"
+          }
```

Comment:
@ryan-blizzard 
the code you have here doesn't run and doesn't pass the tests, please get this working and update this PR